### PR TITLE
166789959 Pagination Support For Articles

### PR DESCRIPTION
**Pull request by kodek-sleuth**
#### What does this PR do?
-Should provide pagination support for articles
#### Description of Task to be completed
- Setup Pagination buttons
#### How should this be manually tested?
Run the following commands in your terminal after cloning the project
```
1. cd ah-92explorers-frontend
2. git checkout ft-paginate-articles-166789959
3. npm i to install all dependencies 
4. npm run dev` and navigate to localhost:8080
```
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166789959](https://www.pivotaltracker.com/story/show/166789959)
#### Screenshots (if appropriate)
Screen Shot 2019-09-03 at 22 12 09
#### Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I don't have more than 3 major commits on this PR

### CODE REVIEW

File: src/components/Articles/Feed.jsx [hunk 1]
```diff
@@ -6,20 +6,40 @@ import Spinner from '../Spinner/Spinner.jsx';
...
+
   render() {
     if (this.props.loading || this.props.articles[0] === undefined) {
       return <Spinner />;
     }
     const { articles } = this.props;
     return (
       <div data-test="homePage" className="feedHome">
-        <Navbar token={articles[14].token} username={articles[14].username} avatar={articles[14].image} />
+        <Navbar token={localStorage.getItem('jwtToken')} username={localStorage.getItem('username')} avatar={localStorage.getItem('image')} />
```
Comment (elemanhillary-zz) [inline src/components/Articles/Feed.jsx:80]: if the token is not there that means line `42` will break the application. Kindly fix this.

File: src/components/Articles/Feed.jsx [hunk 2]
```diff
@@ -190,6 +210,16 @@ export class Feed extends Component {
               </div>
             </div>
           </div>
+          <div className="pagination">
+            <button type="button" className="paginationBtnsIcons" onClick={this.changePaginationLeft}><img src="https://image.flaticon.com/icons/svg/892/892513.svg" alt="" className="paginationIconBack" /></button>
+            <button type="button" value="1" onClick={this.changePage} className="paginationBtns" id="btn1">1</button>
```
Comment (elemanhillary-zz) [inline src/components/Articles/Feed.jsx]: you hardcoded the buttons from `1 to 2` you can make this a 4 liner or 6, kindly revise this and implement it.

**Review by elemanhillary-zz — DISMISSED** [review]
the coverage has dropped and this time around. fix the coverage.

File: src/components/Articles/Feed.jsx (hunk 1 above)
Comment (kodek-sleuth) [reply src/components/Articles/Feed.jsx:80]: Actually test it before you make a conclusion to this assumption

File: src/components/Articles/Feed.jsx (hunk 2 above)
Comment (kodek-sleuth) [reply src/components/Articles/Feed.jsx]: Give me an example implementation of this

Comment (elemanhillary-zz) [reply src/components/Articles/Feed.jsx]:
```javascript
for(let i = 1; i <= 6; i++) <button type="button" value=`${i}` onClick={this.changePage} className="paginationBtns" id=`btn${i}`>i</button>
```

**Review by elemanhillary-zz — CHANGES_REQUESTED** [review]
i have given not a more clean example but rather how to do it

**Review by elemanhillary-zz — DISMISSED** [review]
and coverage has dropped kindly fix that

**Review by Hervera — DISMISSED** [review]
Ready to be merged

**Review by niyoceles — DISMISSED** [review]
ready to be merged.

**Review by IsaiahRn — DISMISSED** [review]
This PR is ready to be merged!

**Review by elemanhillary-zz — DISMISSED** [review]
ready for merge

**Review by elemanhillary-zz — APPROVED** [review]
we can ignore, for now, the difference of `0.3` coverage drop and merge it

**Review by niyoceles — APPROVED** [review]
Even the coverage decrease `0.3`, not more than `1`, The PR can be merged.

**Review by Hervera — APPROVED** [review]

**Review by IsaiahRn — APPROVED** [review]
After reviewing and testing the functionality, this PR can be merged!

**Pull request merged**